_api-reference/cluster-decommission.md

```diff
@@ -0,0 +1,80 @@
+---
+layout: default
+title: Cluster decommission 
+nav_order: 20
+---
+
+# Cluster decommission
+
+The cluster decommission operation adds support decommissioning based on awareness. It greatly benefits uses with multi-zone deployments, where awareness attributes, such as `zones`, can aid in applying new upgrades to a cluster in controlled fashion. This is especially useful during outages, in which case, you can decommission that unhealthy zone to prevent replication requests from getting stalled and prevent your request backlog from becoming to large.
```

Comment:
```suggestion
The cluster decommission operation adds support decommissioning based on awareness. It greatly benefits multi-zone deployments, where awareness attributes, such as `zones`, can aid in applying new upgrades to a cluster in a controlled fashion. This is especially useful during outages, in which case, you can decommission the unhealthy zone to prevent replication requests from stalling and prevent your request backlog from becoming too large.
```

Reply:
"support for decommissioning"?